.github/workflows/codeql-analysis.yml

```diff
@@ -35,11 +35,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'go', 'python' ]
+        language:
+        - name: 'go'
+          path: 'buildpacks'
+        - name: 'python'
+          path: 'invokers/python'
+        - name: 'java'
+          path: 'invokers/java'
         # Re-add Java into main when it works
```

Comment:
Do we want to remove this comment?

Reply:
> LGTM, just remove the comment about Java!

@andrew-su Thanks for the fast review. I'm actually not done with this PR and the above is just an experiment -- meant to put it in draft.

Noted about the java comment though and will remove! 👍